runcellpose.py

```diff
@@ -326,7 +398,7 @@ def run(self, workspace):
         self.add_measurements(workspace)
 
         if self.show_window:
-            if x.volumetric:

```

Comment:
I think we want to keep this as volumetric - because no mater which 3D method we've used, we want to make sure to treat 3D images separately from 2D images for display purposes, yes? Tell me if I'm reading it wrong.